src/vivarium_gates_nutrition_optimization_child/components/population.py

```diff
@@ -132,4 +159,24 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
         population["sex"] = "Female"
         population.loc[population.index % 2 == 1, "sex"] = "Male"
         self.register_simulants(population[list(self.key_columns)])
+        population["subnational"] = self._distribute_subnational_locations(population.index)
```

Comment:
This will work, but why not just do this:
```suggestion
        population["subnational"] = self.location
```
We don't use the subnational for anything anyway.